test/webhooks/service.yaml

```
@@ -0,0 +1,18 @@
+  ports:
+    - name: https
+      port: 443
+      protocol: TCP
+      # targetPort must match the port your webhook server actually listens on
+      # if you haven't overridden it in setup.Options.WebhookServer.Port, the default is 9443
+      targetPort: 9443
+  selector:
+    # your controller-manager (or webhook) usually carries these labels;
+    # swap them out for whatever you’re using in your Deployment/Pod spec
+    control-plane: controller-manager
+    app.kubernetes.io/name: kserve-controller-manager
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)